client/gutenberg/extensions/hello-dolly/hello-block.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import Badge from 'components/badge';
+import Button from 'components/button';
+import Notice from 'components/notice';
```

Comment:
Is this an indication to start pulling those to decoupled modules or something? A first `@calypso/components` dependency starting with these three components?

Reply:
that's the aim eventually. for now we're not using them but soon we'll be importing them directly. maybe we'll add an import alias to import from `@calypso/components` but we're still some ways off from creating packages.

Reply:
See https://github.com/Automattic/wp-calypso/pull/26048#issuecomment-405405486 tho:

> I've updated #24788 and used it to publish `@automattic/simple-components@0.1.0-alpha.7` to include `Badge`, `Button`, and `Notice` components (only transpiled using Babel, but not bundled using webpack). I'd like to try that package in combination with this PR tomorrow :slightly_smiling_face: 

Agree that using those from a block is for a next step tho.